Source/Session.swift

```diff
@@ -174,37 +178,41 @@ open class Session {
                        parameters: Parameters? = nil,
                        encoding: ParameterEncoding = URLEncoding.default,
                        headers: HTTPHeaders? = nil,
+                       interceptor: RequestInterceptor? = nil,
                        to destination: DownloadRequest.Destination? = nil) -> DownloadRequest {
         let convertible = RequestConvertible(url: convertible,
                                              method: method,
                                              parameters: parameters,
                                              encoding: encoding,
                                              headers: headers)
 
-        return download(convertible, to: destination)
+        return download(convertible, interceptor: interceptor, to: destination)
     }
 
     open func download<Parameters: Encodable>(_ convertible: URLConvertible,
                                               method: HTTPMethod = .get,
                                               parameters: Parameters? = nil,
                                               encoder: ParameterEncoder = JSONParameterEncoder.default,
                                               headers: HTTPHeaders? = nil,
+                                              interceptor: RequestInterceptor? = nil,
                                               to destination: DownloadRequest.Destination? = nil) -> DownloadRequest {
```

Comment:
I really struggled with the external API here. Given that all the other parameters are stacked in front of the destination with defaults, I didn't think it made sense to move the `interceptor` after the `destination`, even if it reads better. VERY open to suggestions...

Reply:
I agree, this spot makes sense.